PipeMethodCalls/Endpoints/PipeServerWithCallback.cs

```diff
@@ -35,6 +35,17 @@ public PipeServerWithCallback(string pipeName, Func<THandling> handlerFactoryFun
 			this.options = options;
 		}
 		
+		/// <summary>

```

Comment:
Is there a reason you didn't add these constructors to PipeClient/PipeClientWithCallback?

Reply:
My thinking behind this absence was for the following reasons:
- My understanding was that the server side owns the pipe and as a result you can accomplish _most_ if not all advanced security through the server side, alone.
- Testing my use-case, I found your defaults for the client seem to be very sensible.  And as mentioned above, I didn't see much value in upgrading a client pipe (although I'm very much open to the possibility if I'm missing something).  I was concerned that exposing it left room for issues with a poorly configured client side pipe.

Reply:
I think if we can do the checks mentioned in the other comment we will be OK. It would be odd not to have symmetry here, especially if there's some other setting that gets added to the API and I haven't exposed it yet.

Reply:
Sure, I'll work on getting this put in place.  Thanks for the responsive feedback!

Reply:
This has been implemented